### PR TITLE
00 configure environment

### LINKED ISSUE #43: 00 - Configure environment
- initiate npm 
- install handlebars
- install sass
- install webpack

### PR DESCRIPTION
**Pull request by JoaresMiranda**
Update README and install dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (marceloglacial) [inline package.json]: Move this file to front-end folder. 
All the project dependencies for front-end should live on front-end folder

File: README.md [hunk 2]
```diff
@@ -15,7 +20,15 @@ Website created for assessment of development techniques
 ### Live demo
 - <a href="https://joaresmiranda.github.io/2019-surfshop-poc/front-end/">Live demo</a>
 
+## How to install
+- [Install npm](https://www.npmjs.com/get-npm)
```
Comment (marceloglacial) [inline README.md:22]: Add a step do change folder to front-end

**Review by marceloglacial — CHANGES_REQUESTED** [review]

**Comment by JoaresMiranda** [comment]
Move files to front-end folder and create how to

File: README.md [hunk 3]
```diff
@@ -15,7 +20,39 @@ Website created for assessment of development techniques
 ### Live demo
 - <a href="https://joaresmiranda.github.io/2019-surfshop-poc/front-end/">Live demo</a>
 
-### Screenshots
+## How to install
```
Comment (marceloglacial) [inline README.md:21]: Please update the install  command. Now all the files live on the front-end folder

File: front-end/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (marceloglacial) [inline front-end/package.json:1]: Delete package-lock.json from the **root folder**.

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -9,6 +9,9 @@ Icon
 # Thumbnails
 ._*
 
+# Files
+package-lock.json
```
Comment (marceloglacial) [inline .gitignore]: Remove it from gitignore.

File: .gitignore [hunk 6]
```diff
@@ -9,6 +9,9 @@ Icon
 # Thumbnails
```
Comment (marceloglacial) [inline .gitignore]: Move this file to front-end folder. Remember, all the files live in the front-end folder

**Review by marceloglacial — CHANGES_REQUESTED** [review]

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**